### DISCUSSION TITLE
timezone

### DISCUSSION THREAD

**Discussion by cirrusflyer** [vinceanalytics/vince#21] [General]
Hello, is there a way to set timezone?  I just started using:

       - TZ=America/New_York

But don't think it's working.

**Comment by gernest** [comment]
There is no way to set timezones.

**Reply by cirrusflyer** [reply]
So how do my stats page know what timezone to use?  Or just always uses UTC?  For example, umami allows the user account to choose the timezone so as to display the stats in that timezone.  Can this be a feature request?  Thanks

**Reply by gernest** [reply]
Only UTC is supported.

> Can this be a feature request?

I'm the only dev on vince, I can't take feature requests at the moment. So a honest answer here is No.
